lpr.WebAPI/Application.cs

```diff
@@ -6,59 +6,115 @@
 using lpr.Data.Contexts;
 using lpr.Logic.Services;
 using lpr.WebAPI.Middleware;
-using Microsoft.AspNetCore.Builder;
 using Microsoft.EntityFrameworkCore;
-using Microsoft.Extensions.DependencyInjection;
+using Microsoft.OpenApi.Models;
 
 namespace lpr.WebAPI {
   public class Application {
     public WebApplicationBuilder builder;
     public Application(string[] args) {
-      builder = WebApplication.CreateBuilder(args);
+        builder = WebApplication.CreateBuilder(args);
     }
 
     public void AddDatabaseConnection(string connectionString) {
-      builder.Services.AddDbContext<ILprDbContext, LprContext>(options => {
-        options.UseMySql(connectionString,
-                         new MariaDbServerVersion(new Version(10, 5, 9)));
-      });
-
-      builder.Services.AddScoped<IPackageService, PackageService>();
-          builder.Services.AddScoped<IPackageData, PackageData>();
-
-          using (var scope =
-                     builder.Services.BuildServiceProvider().CreateScope()) {
-            using (var context =
-                       scope.ServiceProvider.GetService<ILprDbContext>()) {
-              context.Database.EnsureCreated();
-            }
-          } builder.Services.AddControllers();
+        builder.Services.AddDbContext<ILprDbContext, LprContext>(options => {
+          options.UseMySql(connectionString,
+                          new MariaDbServerVersion(new Version(10, 5, 9)));
+        });
+
+        builder.Services.AddScoped<IPackageService, PackageService>();
+        builder.Services.AddScoped<IPackageData, PackageData>();
+
+        using (var scope =
+                    builder.Services.BuildServiceProvider().CreateScope()) {
+          using (var context =
```

Comment:
Isnt there a new more eligant way to use "using"?

Reply:
It might be,
The Application.cs file is prune to rapid changes.
So optimizing the file is often quickly discarded.

Reply:
But we'll look into this.